static/js/stream_settings_ui.js

```diff
@@ -1097,10 +1114,14 @@ export function update_stream_privacy_choices(policy) {
 }
 
 export function initialize() {
-    $("#manage_streams_container").on("click", ".create_stream_button", (e) => {
-        e.preventDefault();
-        open_create_stream();
-    });
+    $("#manage_streams_container").on(
+        "click",
+        ".create_stream_button,.create_new_stream_link",
```

Comment:
Why do we need to add `create_new_stream_link` class here, since the `<a>` tag is already linked to "#streams/new" ?